source/common/http/utility.h

```diff
@@ -270,6 +270,7 @@ bool isWebSocketUpgradeRequest(const RequestHeaderMap& headers);
 Http1Settings parseHttp1Settings(const envoy::config::core::v3::Http1ProtocolOptions& config);
 
 struct EncodeFunctions {
+  std::function<void(ResponseHeaderMap& headers)> modify_headers_;
```

Comment:
Oops. Sorry, missed this. I think you need to add comments as well just like the other entries in this struct.

Reply:
Done